predicators/ground_truth_models/sticky_table/options.py

```diff
@@ -30,19 +30,27 @@ def get_options(cls, env_name: str, types: Dict[str, Type],
         PickFromTable = utils.SingletonParameterizedOption(
             # variables: [cube, table]
             "PickFromTable",
-            cls._create_pick_policy(action_space),
+            cls._create_pass_through_policy(action_space),
+            # Parameters are absolute x, y actions.
+            params_space=Box(
+                np.array([StickyTableEnv.x_lb, StickyTableEnv.y_lb]),
+                np.array([StickyTableEnv.x_ub, StickyTableEnv.y_ub])),
             types=[cube_type, table_type])
 
         PickFromFloor = utils.SingletonParameterizedOption(
             # variables: [cube]
             "PickFromFloor",
-            cls._create_pick_policy(action_space),
+            cls._create_pass_through_policy(action_space),
+            # Parameters are absolute x, y actions.
+            params_space=Box(
+                np.array([StickyTableEnv.x_lb, StickyTableEnv.y_lb]),
+                np.array([StickyTableEnv.x_ub, StickyTableEnv.y_ub])),
             types=[cube_type])
 
         PlaceOnTable = utils.SingletonParameterizedOption(
             # variables: [cube, table]
             "PlaceOnTable",
-            cls._create_place_policy(action_space),
+            cls._create_pass_through_policy(action_space),
```

Comment:
wait - why did you rename `_create_place_policy` to `_create_pass_through_policy`? There appears to be no functional change?

Reply:
because it's being used now in the pick option also, not just the place option. you're right that it hasn't functionally changed, it was always just a "pass through" policy